docs/reference/language/modules.md

```diff
@@ -130,7 +130,7 @@ We simplify it by the expression:
 open import Prelude;
 ```
 
-The `hiding` keyword can be used within an `open-import` statement.
+The `hiding` keyword can be used within a `open-import` statement.
```

Comment:
@paulcadman what is the correct article here? Because "open" begins with an "o", so "an" would seem appropriate phonetically. But I'm not sure about the exact rules.

Reply:
Fixed